### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by andpodob**
Hi,
I've added link and icon to github popup (that was nice and easy part of todays work), and carousel with photos and sample text to hobby popup. I've struggled a lot with this second modification because bootstrap scales images by width by default but i needed to scale them by height. I think the overall effect is not bad, take a look.

### CODE REVIEW

**Review by hkveeranki — COMMENTED** [review]
For some, I don't see carousel when I click on hobbies. I see a blank popup.

**Comment by hkveeranki** [comment]
> For some, I don't see carousel when I click on hobbies. I see a blank popup.

I checked out to add-content branch and I can see some sample content and a single image.

**Comment by andpodob** [comment]
If you wait a second image should slide to another one, I've chosen automatic carousel without controls, I can add them if you see it as a better solution. Do I understand correctly that after checking out to add-content hobby and github are not a blank popups?

**Comment by andpodob** [comment]
> If you wait a second image should slide to another one, I've chosen automatic carousel without controls, I can add them if you see it as a better solution. Do I understand correctly that after checking out to add-content hobby and github are not a blank popups?

ok, I launched page once again and I see that controls are definitely necessary, it's a bit non-intuitive without them.

**Comment by andpodob** [comment]
I've added controls to the slider, please check how it works on your side.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -41,6 +41,14 @@
...
+                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Previous</span>
+                </a>
+                <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next" onclick="$('#hobbySlides').carousel('next')">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Next</span>
+                </a>
             </div>
             <p>
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ac lacinia velit. Vivamus tristique lacinia nibh, sed tristique sem. Etiam ultrices sapien quis faucibus malesuada. Morbi hendrerit ut diam sit amet dictum. Aenean turpis nisl, fringilla convallis justo varius, blandit vehicula arcu. Sed viverra elementum odio. Nulla at eros sed massa eleifend pellentesque. Ut dignissim nisl ut viverra dictum. Curabitur id lectus viverra, feugiat libero eget, consequat tortor.
```
Comment (hkveeranki) [inline portfolio/src/main/webapp/index.html:54]: You will add the content in another comment?

**Review by hkveeranki — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (hkveeranki) [reply portfolio/src/main/webapp/index.html:54]: Looks good now. Yeah having buttons is more intuitive.

Comment (andpodob) [reply portfolio/src/main/webapp/index.html:54]: Sure, I'm adding, just added contact data. At the end I will provide some more meaningful text about myself and my hobby, for now only example content to see how it looks like.

**Pull request merged**